starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import Character from './Character';
+import './StarWars.css';
+
+
+const CharacterList = props =>{
+    return(
+        <div className="character-list">
+        {props.starwarsChars.map(character =>(
+            <Character characterArray ={character} />
```

Comment:
Theres an error in your console.log 

> Warning: Each child in a list should have a unique "key" prop.

So you need to add `key={character.created}` after characterArray={character} in order to fix it